opensrp-giz-malawi/src/main/java/org/smartregister/giz/sync/GizMalawiProcessorForJava.java

```diff
@@ -152,6 +160,10 @@ public void processClient(List<EventClient> eventClients) throws Exception {
         }
     }
 
+    private boolean childExists(String entityId) {
+        return ChildLibrary.getInstance().eventClientRepository().checkIfExists(EventClientRepository.Table.client, entityId);
```

Comment:
Use the `GizMalawiApplication` instance here instead to get the `eventClientRepository`

Reply:
cool